blackbox/docs/sql/statements/alter-table.rst

```diff
@@ -170,6 +170,7 @@ where ``reroute_option`` is::
 
     { MOVE SHARD shard_id FROM node_id TO node_id
       | ALLOCATE REPLICA SHARD shard_id ON node_id
+      | PROMOTE REPLICA SHARD shard_id WITH (accept_data_loss = { TRUE | FALSE })
```

Comment:
Not sure if `ON node_id` will be required, I think from a user perspective it would be easier to use without requiring the `node_id`. (Given that the stale replica shard is already assigned to a node in the scenario where this will be used, it should be possible to not require it)

Reply:
Other suggestions so far:

- `ALLOCATE REPLICA SHARD ... WITH (accept_data_loss = true, promote_stale_replica = true)` (would conflict a bit with ommitting the `ON node_id`)
- `PROMOTE TO PRIMARY` instead of `PROMOTE REPLICA` 
- `ALLOCATE PRIMARY SHARD` 

Note that currently `PRIMARY` and `PROMOTE` are no keywords, so we'll be adding new keywords.